src/main/java/com/skytask/channel/ProductProcessor.java

```diff
@@ -0,0 +1,7 @@
+package com.skytask.channel;
+
+import org.springframework.cloud.stream.annotation.EnableBinding;
+
+@EnableBinding(ProductSource.class)
+public class ProductProcessor {
```

Comment:
Could i know what are you using this class for?

Reply:
Yes, following a tutorial I though it was necessary but it is not, I will remove it.

Reply:
Yes, please do, do not add any unnecessary code to your PRs.